airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -476,6 +484,28 @@ public void writeStandardSync(final StandardSync standardSync) throws JsonValida
     return persistence.listConfigs(ConfigSchema.STANDARD_SYNC, StandardSync.class);
   }
 
+  public List<StandardSync> listWorkspaceStandardSyncs(final UUID workspaceId) throws IOException {
+    final Result<Record> result = database.query(ctx -> ctx.select(CONNECTION.asterisk())
+        .from(CONNECTION)
+        .join(ACTOR).on(CONNECTION.SOURCE_ID.eq(ACTOR.ID))
+        .where(ACTOR.WORKSPACE_ID.eq(workspaceId))).fetch();
+    final List<StandardSync> standardSyncs = new ArrayList<>();
+    for (final Record record : result) {
+      final UUID connectionId = record.get(ACTOR.ID);
```

Comment:
It seems like you are setting `connectionId` to the ID of the Actor record here, not the ID of the Connection record. This is a mistake right? If so, why did the unit tests not catch this?

Reply:
You are totally correct! About the unit test, I just double checked and it seems the test fixtures for Sync all have all an empty list of operationsIds. I will improve the test data to be able to catch those cases.

Reply:
Actually I am wrong here, the test data have operationIds

Reply:
Having looked at what was happening here, it seems the code was working correctly since record.get() try to find first an exact full qualified name match on the row and if it does not fall back on trying to find a column with the same name.
So although my code was wrong, it was fetching connection.id from the row anyway.